bin/api.wsgi

```diff
@@ -67,6 +97,8 @@ if not api.app.config['drone_secret']:
 if not os.path.exists(api.app.config['data_path']):
     os.makedirs(api.app.config['data_path'])
 
+# FIXME hard-coded off to fix crash at @gsfr's direction
+centralclient_enabled = False
```

Comment:
I don't currently know how to avoid this crash; if you like we can screen share later to debug.
Alternatively, let me know if you're fine with this merging.